diff --git a/setup.py b/setup.py
index 3bff4f2..a6bbf9b 100644
--- a/setup.py
+++ b/setup.py
@@ -23,31 +23,6 @@ setuptools.setup(
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/DeepLabCut/DeepLabCut",
-    install_requires=[
-        "ipython",
-        "filterpy",
-        "ruamel.yaml>=0.15.0",
-        "imgaug>=0.4.0",
-        "numba",
-        "matplotlib",
-        "networkx",
-        "numpy",
-        "pandas>=1.0.1",
-        "scikit-image>=0.17,<=0.18.1",
-        "scikit-learn",
-        "scipy>=1.4",
-        "statsmodels>=0.11",
-        "tables<=3.6.1",
-        "tensorflow>=2.0",
-        "tensorpack",
-        "tf_slim",
-        "tqdm",
-        "moviepy",
-        "pyyaml",
-        "Pillow>=7.1",
-    ],
-    extras_require={"gui": ["wxpython<4.1"]},
-    scripts=["deeplabcut/pose_estimation_tensorflow/models/pretrained/download.sh"],
     packages=setuptools.find_packages(),
     data_files=[
         (
