diff --git a/deeplabcut/__init__.py b/deeplabcut/__init__.py
index 95caf05..c9ff61a 100644
--- a/deeplabcut/__init__.py
+++ b/deeplabcut/__init__.py
@@ -29,9 +29,7 @@ try:
 
     from deeplabcut.utils.skeleton import SkeletonBuilder
 except (ModuleNotFoundError, ImportError):
-    print(
-        "DLC loaded in light mode; you cannot use any GUI (labeling, relabeling and standalone GUI)"
-    )
+    pass
 
 from deeplabcut.create_project import (
     create_new_project,
