From 4cc8338e44e480855cfdfff5a55ac8843438d88f Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sun, 16 Jun 2024 08:49:29 -0400
Subject: [PATCH 6/8] Remove hard dependency on torch

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0750f614..f2e81267 100644
--- a/setup.py
+++ b/setup.py
@@ -39,7 +39,6 @@ setuptools.setup(
         "scikit-learn>=1.0",
         "scipy>=1.9",
         "statsmodels>=0.11",
-        "torch",
         "tqdm",
         "pyyaml",
         "Pillow>=7.1",
-- 
2.43.2

