From 622109fd243771ecb9510215caa97a636f512a00 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sat, 13 May 2023 13:41:31 -0400
Subject: [PATCH 09/11] Use QRegularExpression

---
 deeplabcut/gui/tabs/modelzoo.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/deeplabcut/gui/tabs/modelzoo.py b/deeplabcut/gui/tabs/modelzoo.py
index 2b61376b..c946769b 100644
--- a/deeplabcut/gui/tabs/modelzoo.py
+++ b/deeplabcut/gui/tabs/modelzoo.py
@@ -10,7 +10,7 @@
 #
 import deeplabcut
 from qtpy import QtWidgets
-from qtpy.QtCore import Qt, Signal, QTimer
+from qtpy.QtCore import Qt, Signal, QTimer, QRegularExpression
 from qtpy.QtGui import QRegularExpressionValidator
 from deeplabcut.gui.components import (
     DefaultTab,
@@ -33,7 +33,7 @@ class RegExpValidator(QRegularExpressionValidator):
 class ModelZoo(DefaultTab):
     def __init__(self, root, parent, h1_description):
         super().__init__(root, parent, h1_description)
-        self._val_pattern = "(\d{3,5},\s*)+\d{3,5}"
+        self._val_pattern = QRegularExpression(r"(\d{3,5},\s*)+\d{3,5}")
         self._set_page()
 
     @property
-- 
2.40.1

