From 2c3b276c9159d779ce2477bf3622c64a196526c6 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sat, 13 May 2023 14:16:32 -0400
Subject: [PATCH 11/14] Use native dialogs when selecting videos

---
 deeplabcut/gui/tabs/create_project.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/deeplabcut/gui/tabs/create_project.py b/deeplabcut/gui/tabs/create_project.py
index 8d0e494d..34febb20 100644
--- a/deeplabcut/gui/tabs/create_project.py
+++ b/deeplabcut/gui/tabs/create_project.py
@@ -109,7 +109,6 @@ class ProjectCreator(QtWidgets.QDialog):
             self,
             "Please select a folder",
             self.loc_default,
-            QtWidgets.QFileDialog.DontUseNativeDialog,
         )
         if not folder:
             return
-- 
2.40.1

